NYPL/dgx-header-component
# SCC-3226: Adding a third radio button for searching

Resolves [SCC-3226](https://jira.nypl.org/browse/SCC-3226)

- This adds the infrastructure code to enable a third radio button for the search form.
- The current mobile search dropdown contains two buttons. This is now updated to include the three radio buttons instead of adding another button. This can be compared on the prod site https://header.nypl.org/.
- This PR points to a feature branch called `third-radio-btn-feature`. This feature won't be released soon so we don't want to merge it into the main `development` and `master` branches just yet.
- The verbiage isn't complete and its addition will be a separate PR.
- ~The URLs for searching aren't complete and their addition will be a separate PR. This means that when searching and selecting "Search the Circulating Catalog" and "Search the Research Catalog", both selections will go to the same site.~ This is no longer the case and selecting "Search the Research Catalog" will go to the Research Catalog site.

## How to test

- Clone the repo and install dependencies
- Make sure you are using node v10
- Run `npm run start:dev` and it should open the dev site in a browser at "localhost:8080".
- Test on mobile and desktop.

Some screenshots
<img width="564" alt="Screen Shot 2022-08-23 at 3 24 34 PM" src="https://user-images.githubusercontent.com/1280564/186250625-92c925a4-85b2-43ea-b29f-8c311dbcd216.png">
<img width="1222" alt="Screen Shot 2022-08-23 at 3 24 25 PM" src="https://user-images.githubusercontent.com/1280564/186250635-bf3734b6-4274-4e2c-9b7d-d3790559612c.png">
